app/src/main/res/values-ar/strings.xml

```diff
@@ -50,11 +50,15 @@
     <string name="others">الاخرين</string>
     <string name="storage_free">free</string>
     <string name="total_storage">إجمالي مساحة التخزين: %s</string>
+    <!-- Password protected files -->
+    <string name="enter_password">Enter password</string>
+    <string name="password">Password</string>
+    <string name="empty_password">Please enter the password</string>
```

Comment:
what error is shown at wrong password?

Reply:
It is currently displayed as a toast using the exception message (the code which was already there). For english it says: `Error: Password required or incorrect password`

Do you want me to add a custom message for that case too?

Reply:
lets actually move these strings into Commons at <!-- Password protection --> section, they could be reused in other apps too